packages/esm-service-queues-app/src/transition-latest-queue-entry/transition-latest-queue-entry.component.tsx

```diff
@@ -1,33 +1,37 @@
 import React from 'react';
 import { useTranslation } from 'react-i18next';
 import { useLatestQueueEntry } from './transition-latest-queue-entry.resource';
+import AddPatientToQueueModal from './add-patient-to-queue-modal/add-patient-to-queue.modal';
 import TransitionQueueEntryModal from '../queue-table/queue-entry-actions/transition-queue-entry.modal';
+import { type Visit } from '@openmrs/esm-framework';
 
-interface TransitionLatestQueueEntryProps {
-  patientUuid: string;
+interface TransitionQueueProps {
+  activeVisit: Visit;
   closeModal: () => void;
-  modalTitle?: string;
 }
 
-const TransitionLatestQueueEntry: React.FC<TransitionLatestQueueEntryProps> = ({
-  closeModal,
-  patientUuid,
-  modalTitle,
-}) => {
+const TransitionQueue: React.FC<TransitionQueueProps> = ({ closeModal, activeVisit }) => {
+  const patientUuid = activeVisit?.patient?.uuid;
   const { t } = useTranslation();
-  const { data: queueEntry, error, isLoading } = useLatestQueueEntry(patientUuid);
-
-  if (error || !queueEntry) {
-    return null;
-  }
+  const { data: queueEntry } = useLatestQueueEntry(patientUuid);
```

Comment:
Sorry, I didn't notice this before, should the `error` and `isLoading` be left in? Possibly to show a loading indicator while its loading, and to display an error if there is one?